Makefile

```diff
@@ -108,3 +109,6 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+certmanager-install:
+	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
```

Comment:
This makes it easier to install everything in an existing cluster.
It's dupliated in the e2e test setup, which is a shame.